src/DynamoCoreWpf/ViewModels/Core/DynamoViewModel.cs

```diff
@@ -922,8 +922,8 @@ internal void CrashGracefully(Exception ex, bool fatal = false)
                 {
                     exceptionAssembly = ex.InnerException?.TargetSite?.Module?.Assembly;
                 }
-                
```

Comment:
do you know why there are too many empty space changes? 
Seems like there are too much changes but most of them are due to the space space removal.

Reply:
Yeah, i noticed those, no idea why they got added, but seems to be correct though, so I did not remove.

Reply:
In my experience if your editor isn't set to strip trailing spaces these creep in without anyone being the wiser. Might make sense to align on whether we make this setting? On Vera we had a linter which would fail on trailing space. That's going a bit far I think, but it did ensure this sort of thing was kept clean.

Reply:
Can you share the link to that linter? Was that at a PR check level?

Reply:
Well, the one I refer to was on TypeScript (tslint). But it looks like this could be accomplished with git hooks: https://stackoverflow.com/questions/591923/make-git-automatically-remove-trailing-white-space-before-committing.

Reply:
We have https://github.com/DynamoDS/Dynamo/blob/master/.editorconfig at repo root. We can suggest everyone to use the editorconfig extension and add all the common rules there.
https://learn.microsoft.com/en-us/visualstudio/ide/create-portable-custom-editor-options